Test/test2/Rlimitouts0.bpl

```diff
@@ -1,5 +1,5 @@
 // We use boogie here because parallel-boogie doesn't work well with -proverLog
-// RUN: %boogie -rlimit:800000 -proverLog:"%t.smt2" "%s"
+// RUN: %boogie -processTimeLimit:0 -timeLimit:0 -rlimit:800000 -proverLog:"%t.smt2" "%s"
```

Comment:
It might be worth having a process time limit here, too, since there's no strict guarantee that Z3 will ever terminate given a resource limit.